System/Units.ecschema.xml

```diff
@@ -479,7 +479,7 @@
     <Unit typeName="IN_HG_AT_60F" phenomenon="PRESSURE" unitSystem="USCUSTOM" definition="PA" numerator="3.37685e3" displayLabel="inHg@60°F" description="See http://physics.nist.gov/cuu/pdf/sp811.pdf Appendix B." />
     <Unit typeName="PA_PER_M" phenomenon="PRESSURE_GRADIENT" unitSystem="SI" definition="PA*M(-1)" displayLabel="Pa/m" />
     <Unit typeName="BAR_PER_KM" phenomenon="PRESSURE_GRADIENT" unitSystem="METRIC" definition="BAR*KM(-1)" displayLabel="bar/km" />
-    <Unit typeName="PERCENT" phenomenon="PERCENTAGE" unitSystem="INTERNATIONAL" definition="ONE" displayLabel="%" description="Unit used to show a ratio as a percentage in the UI."/>
+    <Unit typeName="PERCENT" phenomenon="PERCENTAGE" unitSystem="SI" definition="ONE" displayLabel="%" description="Unit used to show a ratio as a percentage in the UI."/>

```

Comment:
@ColinKerr, Ben had to do this in order to pass schema Validation. What's your take on this one?

Reply:
The persistence unit we decided on is DECIMAL_PERCENT.  so 100% is 1.  With this unit 100% would be 100.

There was push back on listing DECIMAL_PERCENT as SI so we make it INTERNATIONAL and added a rule that allowed it: https://github.com/iTwin/bis-schema-validation/blob/c77f3a4d8247e19f4bd98870b80a0637157bc6f4/bis-rules/src/BisRules.ts#L1045

I suppose this should be better documented or we should change DECIMAL_PERCENT to SI

Reply:
I reverted the change to the unitsschema and switch to the Decimal_Percent unit for the kind of quantity in the railroad units